vignettes/articles/scrubTyphus_example.Rmd

```diff
@@ -123,7 +118,7 @@ Let's also take a look at how antibody responses change by age.
 
 ```{r "plot age"}
 # Plot antibody responses by age
-autoplot(object = xs_data, type = "age-scatter", strata = "Country")
```

Comment:
The `Country` option with capital C throws and error. Replacing it with `country` which the correct case in the dataset.